addons/misc/stringtable.xml

```diff
@@ -1558,6 +1558,7 @@
             <Portuguese>Utilizar kit cirúrgico (corpo inteiro)</Portuguese>
             <Dutch>Gebruik chirurgische kit (volledig lichaam)</Dutch>
             <Italian>Usa il Kit Chirurgico (Corpo Intero)</Italian>
+            <German>Chirurgisches Kit verwenden (Ganzkörper)</German>
```

Comment:
Does not match terminology used in [ACE - Medical](https://github.com/acemod/ACE3/blob/master/addons/medical_treatment/stringtable.xml)